docs/guides/install-python.md

```diff
@@ -1,6 +1,6 @@
 # Installing Python
 
-uv can manage Python installations — Python does not need to be installed to use uv.
+If Python is already installed, uv should [detect and use](#using-an-existing-python-installation) it without configuration. However, uv can also install and manage Python versions.
```

Comment:
I might use "will" instead of "should" here. I think you use "will" below.

Reply:
Thanks!